bokehjs/src/lib/models/tools/inspectors/hover_tool.ts

```diff
@@ -337,7 +344,7 @@ export class HoverToolView extends InspectToolView {
     }
   }
 
-  _render_tooltips(ds: ColumnarDataSource, i: number, vars: any): HTMLElement {
+  _render_tooltips(ds: ColumnarDataSource, i: number | ImageIndex, vars: any): HTMLElement {
```

Comment:
`Vars` was introduced earlier, so perhaps it could be used here.

Reply:
Addressed in 0ac58b267